docs/developer-docs/web-apps/verifiable-credentials/how-it-works.mdx

```diff
@@ -0,0 +1,84 @@
+---
+keywords: [intermediate, concept, identity, credentials, verifiable credentials]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# How it works
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+## Overview
+
+Verifiable credentials (VCs) on ICP rely on Internet Identity (II), which provides different and unlinkable identities for each dapp. A dapp utilizing verifiable credentials can’t ask for a user's credentials from the issuer of the verifiable credential directly. When a relying party wants to obtain a credential from an issuer, it must request it through an identity provider such as Internet Identity.
```

Comment:
Should we introduce any new concepts before we use them in the sentence? For example, maybe we include: "There are four entities participating in the verifiable credentials flow. The first is the user, whom earns a credential, a piece of data that certifies information about the user, from an issuer. An issuer is an entity (a dapp, organization, government, etc.) that verifies information about the user, and provides a relying party (a different dapp, organization, government, etc.) with the certified credential for that user. The identity provider (in this case, Internet Identity) creates a temporary identifier for the user when the issuer passes the credential to the relying party, so that the relying party and the issuer never learn the user's identifiers on the other entity's service."

Reply:
I like it!!

But could we do it in bullets or a paragraph for each concept?

And I would edit the following: "The first is the user, who earns a credential, a piece of data that certifies information about them".

Should this substitute the first paragraph or added?

Reply:
We define the Issuer and Relying Party on this page in the 'Terminology" section on this page, and I would suggest that we add User and Identity provider to this list, then move the 'Terminology' section up higher in the document. I think writing it out in a long paragraph form, such as Mary has suggested, is hard to read and digest.

Reply:
I forgot about that section.

I agree that's a good place to put it.

Could we change the definition of each term with what @marydwyer wrote in the long paragraph?

```
- **User**: Users earn credentials, a piece of data that certifies information about the user, from an issuer.
- **Issuer**: An issuer is an entity (a dapp, organization, government, etc.) that verifies information about the user
- **Relying Party**: A different dapp (organization, government, etc.) which requests certified credentials for that user from the issuer.
- **Identity Provider**: Entity that (in this case, Internet Identity) creates a temporary identifier for the user when the issuer passes the credential to the relying party, so that the relying party and the issuer never learn the user's identifiers on the other entity's service.
```

When you say to move it up, do you mean within the "Overview" page?

Maybe we can link it in this document below the initial paragraph:

```
Check the [terminology]() if a concept is not clear to you.
```

Or something like that.

Reply:
I've revised the terms to use Mary's definitions, and linked to each term in-line ("...credentials from the [issuer](/docs/current/developer-docs/identity/verifiable-credentials/overview#terminology)..."